Zend/zend_object_handlers.c

```diff
@@ -275,14 +275,12 @@ static ZEND_COLD zend_never_inline void zend_bad_property_name(void) /* {{{ */
 }
 /* }}} */
 
-static ZEND_COLD zend_never_inline void zend_forbidden_dynamic_property(
-		zend_class_entry *ce, zend_string *member) {
+ZEND_API zend_never_inline void zend_forbidden_dynamic_property(zend_class_entry *ce, zend_string *member) {
```

Comment:
Missed this earlier - would recommend changing this back to the original `static` declaration since adding this to the public api isn't needed by the functionality in this PR. 

```
Zend/zend_exceptions.c:169:ZEND_API ZEND_COLD void zend_throw_exception_internal(zend_object *exception) /* {{{ */
Zend/zend_exceptions.h:44:ZEND_API ZEND_COLD void zend_throw_exception_internal(zend_object *exception);
```

1. ZEND_API and ZEND_COLD can be combined
2. Functions annotated with ZEND_API are declared in the corresponding headers so that modules (including PECLs) can make use of them
3. **Change this back?** Nothing else outside this file ended up using zend_forbidden_dynamic_property after the last refactoring of the unserializer, PECL authors can easily duplicate this functionality and add context to the error string if useful

Reply:
Yes, I also didn't catch this. I think getting rid of the unnecessary changes is the right way to go.